examples/otelcollector/README.md

```diff
@@ -0,0 +1,30 @@
+# Grafana Agent Pull Mode Integration
+
+This example demonstrates how you can use Grafana Agent with Grafana Pyroscope to profile otel collector.
+
+### 1. Configure Grafana agent
+
+You'll need to configure the Grafana agent for things like profiling configuration, targets, and possibly authentication in order to have the Grafana Agent pull profiles from your application.
+
+You can find a list of [arguments](https://grafana.com/docs/agent/next/flow/reference/components/pyroscope.scrape/#arguments) and [supported blocks](https://grafana.com/docs/agent/next/flow/reference/components/pyroscope.scrape/#blocks) in the [Grafana Agent documentation for pyroscope](https://grafana.com/docs/agent/next/flow/reference/components/pyroscope.scrape/)
```

Comment:
The links in your content point to the /next branch. Should this be /next or /latest? 
```suggestion
You can find a list of [arguments](https://grafana.com/docs/agent/next/flow/reference/components/pyroscope.scrape/#arguments) and [supported blocks](https://grafana.com/docs/agent/next/flow/reference/components/pyroscope.scrape/#blocks) in the [Grafana Agent documentation for Pyroscope](https://grafana.com/docs/agent/next/flow/reference/components/pyroscope.scrape/)
```